portals/publisher/src/main/webapp/source/src/app/components/Apis/Details/Endpoints/endpointUtils.js

```diff
@@ -190,6 +190,12 @@ function createEndpointConfig(endpointType) {
             tmpEndpointConfig.production_endpoints = { config: null, url: 'http://localhost' };
             tmpEndpointConfig.sandbox_endpoints = { config: null, url: 'http://localhost' };
             break;
+        case 'INLINE': // mock implementation is functionally prototype.
```

Comment:
The intended endpointType for this scenario should be 'prototyped'. Therefore, the fix should involve updating the code to reflect this behavior, rather than adding a special case to pass the incoming value.
Which means, the actual issue arises before coming to this level.